packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -0,0 +1,284 @@
+          nextCol = 0;
+
+          if (event.ctrlKey) {
+            // Ctrl + Cmd + Left = Ctrl + Home
+            nextRow = WorklistDataGridUtils.FIRST_NON_HEADER_ROW;
+          }
+        } else {
+          // Left key
+          nextCol -= 1;
+        }
+        break;
+      case KeyCode.KEY_RIGHT:
+        if (event.metaKey) {
+          // Cmd + Right = End
+          nextCol = customProps.columnsLength - 1;
+
+          if (event.ctrlKey) {
+            // Ctrl + Cmd + Right = Ctrl + End
+            nextRow = customProps.rowsLength;
+          }
+        } else {
+          // Right key
+          nextCol += 1;
+        }
+        break;
+      case KeyCode.KEY_HOME:
+        nextCol = 0;
+        if (event.ctrlKey) {
+          nextRow = WorklistDataGridUtils.FIRST_NON_HEADER_ROW; // Assumption is that the first row is the column Heading.
+        }
+        break;
+      case KeyCode.KEY_END:
+        nextCol = customProps.columnsLength - 1; // Col are zero based.
+        if (event.ctrlKey) {
+          // Though rows are zero based, the header is the first row so the rowsLength will
+          // always be one more than then actual number of data rows.
+          nextRow = customProps.rowsLength;
+        }
+        break;
+      case KeyCode.KEY_SPACE:
+        // Default behavior of scrolling by space key needs to be prevented.
+        event.preventDefault();
+        return;
+      case KeyCode.KEY_ESCAPE:
+        if (!isRowSelectionModeEnabled) {
+          if (onCellSelectionChange) {
+            onCellSelectionChange(null, null, coordinates);
+          }
+        }
+        event.preventDefault();
+        break;
+      case KeyCode.KEY_RETURN: // May need to handle KeyCode.KEY_FIREFOX_ENTER
+        if (!isRowSelectionModeEnabled) {
+          if (onNavigationModeChange) {
+            onNavigationModeChange(false);
+          }
+
+          // TODO - Enter: Disable grid navigation.
+          // Enter cell, focus first focusable element, subsequent tab presses cycle between focusable elements within cell.
+          // Do not clear selection. If cell is text only, nothing happens; maintain grid navigation and focus on the cell.
+        }
+        return;
+      default:
+        return;
+    }
+    if (nextRow > customProps.rowsLength || nextCol >= customProps.columnsLength) {
+      event.preventDefault();
+      return;
+    }
+    if (nextCol < 0 || nextRow < 0) {
+      event.preventDefault();
+      return;
+    }
+    if (onMoveCellFocus) {
+      onMoveCellFocus(coordinates, { row: nextRow, col: nextCol });
+    }
+    event.preventDefault();
+  };
+
+  const CellTag = isRowHeader ? 'th' : 'td';
+  return (
+    // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
+    <CellTag
+      key={`${rowId}_${columnId}`}
+      id={`${rowId}_${columnId}`}
+      aria-selected={isSelected ? true : undefined}
+      aria-label={ariaLabel}
+      tabIndex={acceptsFocus ? 0 : -1}
```

Comment:
When would acceptsFocus be false? All cells should be able to receive keyboard focus in some form or another.